pkg/identityserver/config.go

```diff
@@ -121,6 +121,12 @@ type Config struct {
 		TenantID string          `name:"tenant-id" description:"Tenant ID"`
 	} `name:"network"`
 	TelemetryQueue telemetry.TaskQueue `name:"-"`
+	Pagination     Pagination          `name:"pagination" description:"Pagination settings"`
+}
+
+// Pagination default limit.
+type Pagination struct {
+	DefaultLimit uint32 `name:"default-limit" description:"The default limit applied to paginated requests if not specified"`
 }
 
```

Comment:
We generally embed this struct into the IS configuration.

Reply:
I agree with Krishna that it is probably best to remain with what is generally established.

It doesn't make much of a difference as the only impact of creating its own struct is the increase on the amount of types exported by the `identityserver` package. The benefit being the GoDoc reference, which is quicker to check when trying to use a Config's field while developing something.